kind/cmd/kind/kind.go

```diff
@@ -45,23 +44,19 @@ func NewCommand() *cobra.Command {
 
 // Run runs the `kind` root command
 func Run() error {
-	// Trick to avoid glog's 'logging before flag.Parse' warning
-	flag.CommandLine.Parse([]string{})
-	// glog logs to files by default, grr
-	flag.Set("logtostderr", "true")
-
-	cmd := NewCommand()
-	// glog registers global flags on flag.CommandLine...
-	cmd.Flags().AddGoFlagSet(flag.CommandLine)
-
-	// actually execute the cobra commands now...
-	return cmd.Execute()
+	return NewCommand().Execute()
 }
 
 // Main wraps Run, adding an Exit(1) on error
 func Main() {
+	// this formatter is the default, but the timestamps output aren't
+	// particularly useful, they're relative to the command start
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:   true,
+		TimestampFormat: "15:04:05-0700",
+	})
 	if err := Run(); err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
 		os.Exit(1)
```

Comment:
fatal already calls `os.Exit(1)` I think

Reply:
thanks, missed that, that was the intention

Reply:
fixed